apps/console/src/features/authentication-flow-builder/components/script-editor-panel/script-editor-panel.tsx

```diff
@@ -17,12 +17,15 @@
  */
 
 import { DiffOnMount, Monaco, MonacoDiffEditor } from "@monaco-editor/react";
+import Modal from "@mui/material/Modal";
```

Comment:
Shall we import this from oxygen-ui?

Reply:
https://github.com/wso2/identity-apps/pull/5364#discussion_r1468807988

Added a comment here. These needs to be wrapped in Oxygen. Will send a separate PR.